NOTICE

```diff
@@ -3,276 +3,3 @@ Copyright 2017-2018 The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
-
-================================================================
-The Apache License Version 2.0
-================================================================
-The following components are provided under the  Apache License Version 2.0  (http://www.apache.org/licenses/). See project link for details.
-
-github.com/astaxie/beego
-github.com/cockroachdb/cmux
-github.com/coreos/etcd
-github.com/coreos/go-semver/semver
-github.com/coreos/go-systemd/journal
-github.com/coreos/pkg/capnslog
-google.golang.org/genproto/googleapis/rpc/status
-google.golang.org/grpc
-gopkg.in/yaml.v2
-github.com/google/btree
-github.com/grpc-ecosystem/go-grpc-prometheus
-github.com/jonboulle/clockwork
-github.com/matttproud/golang_protobuf_extensions/pbutil
-github.com/prometheus/client_golang/prometheus
-github.com/prometheus/client_model/go
-github.com/prometheus/common
-github.com/prometheus/procfs/xfs
-github.com/ServiceComb/paas-lager
-
-================================================================
-The MIT License
-================================================================
-
-The following components are provided under the  MIT License  (http://www.opensource.org/licenses/mit-license.php). See project link for details.
-
-github.com/beorn7/perks/quantile
-github.com/boltdb/bolt
-github.com/satori/go.uuid
-github.com/rs/cors
-AngularJS v1.6.8 (http://angularjs.org)
-Bootstrap v3.3.7 (http://getbootstrap.com)
-Chart.js v2.7.0 (https://github.com/chartjs/Chart.js)
-jquery v3.2.1 (https://github.com/jquery/jquery)
-JSZip v3.1.4 (https://github.com/Stuk/jszip)
-yaml.js v0.1.5 (https://github.com/jeremyfa/yaml.js)
-js-yaml v3.10.0 (http://github.com/nodeca/js-yaml)
-FileSaver.js v1.1.20151003 (http://github.com/eligrey/FileSaver.js)
-angular-yamljs (http://github.com/mmumshad/angular-yamljs)
```

Comment:
How about the frontend dependencies license which we will be using with cdn servers, where do we need to keep them.

Reply:
We do not redistribute them so we don't need to mention it in either LICENSE or NOTICE.
As per the apache policy, only the bits that is actually bundled in the distribution need to be covered by the LICENSE file.

Reply:
cool, then we don't need any License or NOTICE in frontend as all the dependencies are handled by CDN and no depedencies are bundled together in the release